### ISSUE TITLE
Fresh master clone can't be used

### ISSUE THREAD

**Issue by ignasi35** [lagom/online-auction-java#115]
A fresh clone of `master` is not runnable:

1. clone
2. `sbt runAll`
3.  create a user

**Expected** 

Default user home is displayed

**Actual**

```

[warn] o.a.k.c.NetworkClient - Error while fetching metadata with correlation id 1 : {item-ItemEvent=LEADER_NOT_AVAILABLE}
[warn] o.a.k.c.NetworkClient - Error while fetching metadata with correlation id 2 : {item-ItemEvent=LEADER_NOT_AVAILABLE}
[warn] o.a.k.c.NetworkClient - Error while fetching metadata with correlation id 1 : {bidding-BidEvent=LEADER_NOT_AVAILABLE}
[info] o.h.v.i.u.Version - HV000001: Hibernate Validator 5.2.4.Final
[info] Service webGateway listening for HTTP on 0:0:0:0:0:0:0:0:60496
[info] Service searchImpl listening for HTTP on 0:0:0:0:0:0:0:0:55320
[info] Service itemImpl listening for HTTP on 0:0:0:0:0:0:0:0:53203
[info] Service transactionImpl listening for HTTP on 0:0:0:0:0:0:0:0:51742
[info] Service biddingImpl listening for HTTP on 0:0:0:0:0:0:0:0:58427
[info] Service userImpl listening for HTTP on 0:0:0:0:0:0:0:0:56181
[info] (Services started, press enter to stop and go back to the console...)
[error] application - 

! @74f86hh4b - Internal server error, for (GET) [/my/items/created] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[CompletionException: java.util.NoSuchElementException: No value present]]
        at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:293)
        at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:220)
        at play.api.GlobalSettings$class.onError(GlobalSettings.scala:160)
        at play.api.DefaultGlobal$.onError(GlobalSettings.scala:188)
        at play.api.http.GlobalSettingsHttpErrorHandler.onServerError(HttpErrorHandler.scala:100)
        at play.core.server.netty.PlayRequestHandler$$anonfun$2$$anonfun$apply$1.applyOrElse(PlayRequestHandler.scala:100)
        at play.core.server.netty.PlayRequestHandler$$anonfun$2$$anonfun$apply$1.applyOrElse(PlayRequestHandler.scala:99)
        at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:346)
        at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:345)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
Caused by: java.util.concurrent.CompletionException: java.util.NoSuchElementException: No value present
        at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:273)
        at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:280)
        at java.util.concurrent.CompletableFuture.biApply(CompletableFuture.java:1107)
        at java.util.concurrent.CompletableFuture$BiApply.tryFire(CompletableFuture.java:1070)
        ... 5 common frames omitted
Caused by: java.util.NoSuchElementException: No value present
        at java.util.Optional.get(Optional.java:135)
        at views.html.myItems_Scope0$myItems_Scope1$myItems.apply(myItems.template.scala:41)
        at views.html.myItems_Scope0$myItems_Scope1$myItems.render(myItems.template.scala:110)
        at views.html.myItems.render(myItems.template.scala)
        at controllers.ProfileController.lambda$null$0(ProfileController.java:41)
        at java.util.concurrent.CompletableFuture.biApply(CompletableFuture.java:1105)
        at java.util.concurrent.CompletableFuture$BiApply.tryFire(CompletableFuture.java:1070)
        at java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:443)
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
        at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
```

**Comment by TimMoore** [comment]
@lakhina would you be able to look at this today? Thanks!

**Comment by lakhina** [comment]
@TimMoore Sure. I will.

**Comment by lakhina** [comment]
@TimMoore  I have opened a new PR solving this [issue.](https://github.com/lagom/online-auction-java/pull/118) Please review.

**Comment by TimMoore** [comment]
Great, thanks for fixing @lakhina and thanks for reviewing and merging @yg-apaza!

For future reference, you can add a comment like "Fixes #115" in the description of the pull request. Then, when the pull request is merged, it will close the issue automatically.
